src/main/java/com/teamwizardry/librarianlib/features/animator/Animation.kt

```diff
@@ -7,13 +7,24 @@ import com.teamwizardry.librarianlib.features.kotlin.clamp
  */
 abstract class Animation<T : Any>(val target: T, val property: AnimatableProperty<T>) {
 
+    /**
+     * Default: false
+     *
+     * If this flag is set the start time will be treated as an absolute time, as opposed to being treated as an offset
+     * from the current [Animator] time.
+     */
+    var useAbsoluteTime = false
+
     /**
      * Default: true
      *
      * If this flag is set the start time will be treated as relative to the [Animator] time it is being added to. If it
      * is false, the start time will be unaffected by the Animator's time.
      */
-    var isTimeRelative = true
+    @Deprecated("Use `useAbsoluteTime`", replaceWith = ReplaceWith("!useAbsoluteTime"))
```

Comment:
Did you check this replaceWith rule works properly for both get & set usage?
If not, better to not set it (you don't want automatic replacement to break your code)